### DISCUSSION TITLE
Support source tags of html audio tag

### DISCUSSION THREAD

**Discussion by markusweigelt** [katspaugh/wavesurfer.js#2781] [Ideas]
Instead of the `src` attribute it is possible to define different sources for the same audio file but with different mime types.

```
<audio controls>
  <source src="horse.ogg" type="audio/ogg">
  <source src="horse.mp3" type="audio/mpeg">
  Your browser does not support the audio tag.
</audio>
```
https://www.w3schools.com/tags/tag_audio.asp

The browser takes the appropriate mimetype and plays the audio file. So it would be nice to get the appropriate audio file for current browser from the source tag and load it.

**Comment by katspaugh** [comment]
Looks like this can be easily fixed by using the currentSrc property instead of src.

**Reply by markusweigelt** [reply]
Yes in my case i added following workaround but it would be nice when wavesurfer support this out of the box.

```
const audio = document.getElementsByTagName("audio")[0];
audio.src = audio.currentSrc;
wavesurfer.load(audio);
```

**Reply by katspaugh** [reply]
Fixed here: https://github.com/katspaugh/wavesurfer.ts/pull/84

**Reply by markusweigelt** [reply]
Nice thank you for this fix in rocket speed. 🚀
